CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,43 @@
+# Code of conduct 
```

Comment:
I can't say it's terribly necessary considering the amount of contributions but I don't see anything wrong with the content. Do keep it in, it's aligned with NCC's values.